### PR TITLE
Feature/hii 53

### PR DESCRIPTION
**Pull request by n3op2**
## What

- added hydrogens drop down
- dynamic list of hydrogens
- updated to hii.json to avoid over usage of computing power and power!

#### screenshots
Screenshot 2023-01-17 at 09 17 57
Screenshot 2023-01-17 at 09 16 11
Screenshot 2023-01-17 at 09 15 48
Screenshot 2023-01-17 at 09 15 43
Screenshot 2023-01-17 at 09 15 31
Screenshot 2023-01-17 at 09 15 27

### CODE REVIEW

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
This will conflict with the changes in #35 and also complicate the export process. Is the change to the json just the splitting on `;`? If so we're probably better off doing this in code as otherwise it will complicate export

File: src/assets/hii.json [hunk 1]
```diff
@@ -1,1255 +1,1255 @@
 {
```
Comment (mattdean-digicatapult) [inline src/assets/hii.json]: lot of diff churn on this file

**Review by mattdean-digicatapult — COMMENTED** [review]
Also I'd like to get #35 in first which will affect this PR slightly. This looks good though

File: src/assets/hii.json (hunk 1 above)
Comment (n3op2) [reply src/assets/hii.json]: resolved, not sure why tho, copied from github main

File: src/pages/Home.js [hunk 2]
```diff
@@ -28,6 +28,20 @@ import geojson from '../assets/hii.json'
...
+// give each feature an id and convert hydrogens string into an array
+geojson.features = geojson.features.map((f) => {
+  return {
+    ...f,
+    properties: {
+      ...f.properties,
+      'Type of Hydrogen': f.properties['Type of Hydrogen']
+        .split(';')
+        .map((type) => type.trim()),
+      id: uuid(),
```
Comment (mattdean-digicatapult) [inline src/pages/Home.js]: remove, json now includes id

Comment (n3op2) [reply src/pages/Home.js]: removed

**Review by mattdean-digicatapult — APPROVED** [review]
lgtm 👍

**Pull request merged**